src/shared/views/map/MapTabs/MapTabs.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Tab, Tabs } from 'react-toolbox'
+
+import { MapTab } from '../index'
+
+class MapTabs extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { index: 0 }
+  }
+
+  handleTabChange(index) {
+    this.setState({ index })
+  }
+
+  render() {
+    return (
+      <Tabs index={this.state.index} onChange={this.handleTabChange.bind(this)}>
```

Comment:
Local binding of methods should be avoided. They are used in examples for simplicity but cause the react to create anonymous function every time there is a re-render.

If you instead have a method attached to the component, it will create the function once and keep it in memory.